modules/core/src/bitgo.ts

```diff
@@ -1045,8 +1046,8 @@ export class BitGo {
 
     // BIP32 derivation path is applied to both client and server master keys
     const derivationPath = responseBody.derivationPath;
-    const clientDerivedNode = hdPath(clientHDNode).derive(derivationPath);
-    const serverDerivedNode = hdPath(serverHDNode).derive(derivationPath);
+    const clientDerivedNode = clientHDNode.derivePath(sanitizeLegacyPath(derivationPath));
+    const serverDerivedNode = serverHDNode.derivePath(sanitizeLegacyPath(derivationPath));
```

Comment:
You can call `sanitizeLegacyPath` once and make a `sanitizedDerivationPath` variable right above, saves a call and will make these lines a bit more readable.

Reply:
true, I will revisit this anyway